charts/exposecontroller/templates/rolebinding.yaml

```diff
@@ -10,4 +10,19 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: {{ .Chart.Name }}
-  namespace: {{ .Release.Namespace }}
\ No newline at end of file
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  name: {{ .Chart.Name }}
+  # This only grants permissions within the "default" namespace.
+  namespace: default
```

Comment:
Same as the other comment - we avoid specifying namespaces.  If there are specific roles for environments you require then best to apply them outside of the exposecontroller chart.